lib/nexthop.h

```diff
@@ -86,6 +86,8 @@ struct nexthop {
 					  * active one
 					  */
 #define NEXTHOP_FLAG_RNH_FILTERED  (1 << 5) /* rmap filtered, used by rnh */
+#define NEXTHOP_FLAG_HAS_BACKUP (1 << 6)    /* Backup nexthop index is valid */
```

Comment:
Every nexthop _has_ the cell - but only in some is the value meaningful (or valid). I think I used 'valid' because it fit the line length, but 'meaningful' makes the line too long? I could say '...index is used', I guess - would that be better?

Reply:
Yea that would be better. The comment is a little confusing to someone not knowing what the flag is for.

Reply:
I'll clarify this comment and add a bit to the one with the actual index cell in `struct nexthop` too